modules/olm-arch-catalog-operator.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding_olm/olm-arch.adoc
+
+[id="olm-arch-catalog-operator_{context}"]
+= Catalog Operator
+
+The Catalog Operator is responsible for resolving and installing CSVs and the
+required resources they specify. It is also responsible for watching
+CatalogSources for updates to packages in channels and upgrading them
+(optionally automatically) to the latest available versions.
+
+A user who wishes to track a package in a channel creates a Subscription
+resource configuring the desired package, channel, and the CatalogSource from
+which to pull updates. When updates are found, an appropriate InstallPlan is
+written into the namespace on behalf of the user.
+
+Users can also create an InstallPlan resource directly, containing the names of
```

Comment:
This part is no longer true (it was a long time ago). We are working on new apis that will allow you to specify a bundle directly, but right now you have to go through a subscription, or directly create the CSV.

Reply:
Paragraph removed.